### ISSUE TITLE
Footer

### ISSUE THREAD

**Issue by dan9d9** [chingu-voyages/v11-toucans-team-13#23]
Their footer has a lot of things going on. Maybe we start with a basic footer and add complexity if we have the time.

**Comment by MinervaLong** [comment]
Yes, may be connect with us section to put social media icons, the about section and contact support since it's important client have some way to ask questions
